CharlieBackend.Business/Services/StudentGroupService.cs

```diff
@@ -47,6 +47,20 @@ public async Task<StudentGroupModel> CreateStudentGroupAsync(CreateStudentGroupM
                     }
                 }
 
+                if(studentGroupModel?.MentorIds.Count != 0)
```

Comment:
besides studentGroupModel, MentorIds could also be null. It is better to check studentGroupModel at the beggining of the method, and throw ArgumentNullException.

Reply:
But in type of studentGroupModel (CreateStudentGroupModel) we have data attribute field [Required] for MentorIds